specification/src/chapter2.adoc

```diff
@@ -777,9 +864,9 @@ permission
 
 Isolation can also extend to other features, such as interrupts and debug.
 
-==== Data sealing
+==== Sealing
 
-Sealing is the process of protecting confidential data on a system.
+Sealing is the process of protecting confidential assets on a system.
```

Comment:
```suggestion
Sealing is the process of protecting confidential assets on a system, typically by cryptographically binding sealing keys to the measured state of the workload that owns the assets.
```

Reply:
Different forms of sealing are defined next in this section.

Reply:
that maybe but this introductory definition seems very incomplete.

Reply:
I see the problem now, this section is a little out of kilter with the rest. I have fixed it, and added the measurement based use case as well (already had that elsewhere).